tests/system/base.py

```diff
@@ -0,0 +1,186 @@
+import os
+import pathlib
+import pytest
+import yaml
+
+from mlrun import get_run_db, mlconf
+from mlrun.utils import create_logger
+
+
+logger = create_logger(level='debug', name='test')
+
+
+class TestMLRunSystem:
+
+    root_path = pathlib.Path(__file__).absolute().parent.parent.parent
+    env_file_path = root_path / 'tests' / 'system' / 'env.yml'
+    results_path = root_path / 'tests' / 'test_results' / 'system'
+    mandatory_env_vars = [
+        'MLRUN_DBPATH',
+        'V3IO_API',
+        'V3IO_USERNAME',
+        'V3IO_PASSWORD',
+        'V3IO_ACCESS_KEY',
+    ]
+
+    def setup_method(self, method):
+        self._logger = logger
+        self._logger.info(
+            f'Setting up test {self.__class__.__name__}::{method.__name__}'
+        )
+        self._test_env = {}
+        self._old_env = {}
+        self._setup_env(self._get_env_from_file())
+        mlconf.dbpath = self._test_env['MLRUN_DBPATH']
```

Comment:
add comment explaining why you do it (now I'm remembering I forgot to explain it to you, will explain it offline)